memcached.py

```diff
@@ -5,23 +5,29 @@
 # Copyright (c) 2016, InnoGames GmbH
 #
 
+from argparse import ArgumentParser
 import re
-import socket
-import sys
 import telnetlib
-import time
+from time import time
 
 
-def main(host='127.0.0.1', port='11211'):
-    hostname = socket.gethostname().replace('.', '_')
-    ts = str(int(time.time()))
-    template = 'servers.' + hostname + '.software.memcached.{1} {2} ' + ts
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='memcached')
+    parser.add_argument('host', default='127.0.0.1')
```

Comment:
It is okay, but I don't think it is a good style in this particular case.  I just wanted to make it backwards-compatible.  We can change this later.